src/utils/util.js

```diff
@@ -54,6 +54,16 @@ export function isSetsEqual (a, b) {
     return a.size === b.size && [...a].every(value => b.has(value));
 }
 
+export function equalArrays (m1, m2) {
+    if (m1 && m2 && m1.length === m2.length) {
+        let ret = true;
+        for (let i = 0; i < m1.length; i++) {
+            ret &= m1[i] === m2[i];
```

Comment:
this would be simpler and faster without the ret variable, early returning a false when the values mismatch

Reply:
Yes, I think I changed it already :thinking: